articles/tutorials/building_2d_games/04_working_with_textures/index.md

```diff
@@ -0,0 +1,344 @@
+
+```cs
+_spriteBatch.Draw(_logo, new Vector2(Window.ClientBounds.Width, Window.ClientBounds.Height) * 0.5f, Color.White);
+```
+
+> [!TIP]  
+> In the example above, we multiply the [Vector2](xref:Microsoft.Xna.Framework.Vector2) created by `0.5f` to half the value instead of dividing it by `2.0f`. If you're not used to seeing this, it might seem strange at first, but it's actually an optimization technique. CPUs are able to perform multiplication operations much faster than division operations and reading `* 0.5f` is easily understood to be the same thing as `/ 2.0f` when reading.
+
+We're now telling the position to be a half the width and height of the client bounds. If we run the game now, the logo should be drawn at the center of the game window correct? Well, not exactly. If you run the game now, it will look similar to the following:
+
+<figure><img src="./images/logo-off-center.png" alt="Figure 4-8: Attempting to draw the MonoGame logo centered on the game window."><figcaption><p><strong>Figure 4-8: Attempting to draw the MonoGame logo centered on the game window.</strong></p></figcaption></figure>
+
+So what happened here? Recall from the table above that the *position* parameter is the xy-coordinate position to draw the texture at, with the texture's origin being the top-left corner of the texture. This means, that the image will be drawn at that position starting with the top-left corner of the image. So while we are drawing the image at the center of the game window, the image itself is not centered.
+
+One way to correct this is to subtract half the width and height of the logo image from the game window center position like so:
+
+```cs
+_spriteBatch.Draw(
+  _logo,        // texture
+  new Vector2(  // position
+    (Window.ClientBounds.Width * 0.5f) - (_logo.Width * 0.5f),
+    (Window.ClientBounds.Height * 0.5f) - (_logo.Height * 0.5f)),
+  Color.White); // color
+```
+
+This offsets the position so that it correctly centers the image to the game window.
+
+<figure><img src="./images/logo-centered.png" alt="Figure 4-9: The MonoGame logo drawn centered on the game window."><figcaption><p><strong>Figure 4-9: The MonoGame logo drawn centered on the game window.</strong></p></figcaption></figure>
+
+While this works, this may not be the best way to do it. Let's explore why. The [SpriteBatch.Draw](xref:Microsoft.Xna.Framework.Graphics.SpriteBatch.Draw(Microsoft.Xna.Framework.Graphics.Texture2D,Microsoft.Xna.Framework.Vector2,Microsoft.Xna.Framework.Color)) method contains other overloads that accept more parameters. Let's change the [SpriteBatch.Draw](xref:Microsoft.Xna.Framework.Graphics.SpriteBatch.Draw(Microsoft.Xna.Framework.Graphics.Texture2D,Microsoft.Xna.Framework.Vector2,Microsoft.Xna.Framework.Color)) method call to use the one with all parameters. Adjust the code to the following:

```

Comment:
Worth injecting a note here about why it might "seem" complex, but this is because this is how graphics cards think.  Other engines might "work this out for you", but then it just adds complexity when you want to do it differently.  MonoGame makes this much simpler and allows you to draw your own way!

Reply:
Wording on this section has been reworked to make it clearer and promote the advantages using MonoGame.